src/RBCXManager.cpp

```diff
@@ -231,6 +237,21 @@ bool Manager::printTasksDebugInfo() {
 }
 #endif
 
+void Manager::fault(CoprocStat_FaultStat faultStat) {
+    switch (faultStat.which_fault) {
+    case CoprocStat_FaultStat_oledFault_tag:
+        printf("Oled not connected\n");
```

Comment:
nitpick: Asi by mělo být `ESP_LOGE(TAG, "Oled not connected\n");`, koukám teda, že mám nahoře taky printf - předělej ho prosím taky.